src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] nodesArray;
+    private int capacity;
+    private int size;
+
+    public MyHashMap() {
+        nodesArray = new Node[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+    }
+
+    private int getHash(Object key) {
```

Comment:
yuo can change this method name 
you get index of bucket ?

Reply:
But in HashMap Node has hash not index